Octokit.Reactive/Clients/IObservableMigrationClient.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Octokit.Reactive
+{
+    public interface IObservableMigrationClient
+    {
+        /// <summary>
+        /// A client for GitHub's Migrations API
+        /// </summary>
+        /// <remarks>
+        /// See the <a href="https://developer.github.com/v3/migration/migrations/">Enterprise License API documentation</a> for more information.
+        /// </remarks>
+        IObservableMigrationsClient Migration { get; }
```

Comment:
This property name should be `Migrations` (this will fix the failing convention test which ensures that all properties/methods on the normal interfaces are also present on the observable interfaces.  Currently it thinks `Migrations` is missing, because it's called `Migration` here but `Migrations` on the normal client interface...